### ISSUE TITLE
"Flip" view navigation button

### ISSUE THREAD

**Issue by Keavon** [GraphiteEditor/Graphite#1722]
Our navigation buttons...

![image](https://github.com/GraphiteEditor/Graphite/assets/4388688/bdab9b1b-b11d-45e9-8d33-293712dbaa73)

...should have a quick "flip" checkbox to reverse the view (and a keyboard shortcut). This is useful when painting to see things from a mirrored perspective.

A clear visual indicator is needed to make it obvious that it's active (so this is soft-blocked on @Keavon for UI and icon design).

**Comment by haikalvidya** [comment]
This feature is just flip the view board, ya?, if we export the art board, it will export back on original side?

**Comment by jacinyan** [comment]
Hi @Keavon, I'm interested in working on this. Could you point me to which core module this flip view feature should be implemented in? （Looks like it's do with navigation but just want to make sure I start in the right direction)

**Comment by jacinyan** [comment]
Hi @Keavon, I'm interested in working on this. Could you point me to which module this should be implemented in? (Looks like it's in editor, but just want to make sure)

**Comment by Keavon** [comment]
@jacinyan Yes, in the editor module. You'll want to trace through the code relating to the viewport navigation and specifically the tilt, which is the closest to flip. You can start tracing from an entry point such as `navigate_tool.rs` which deals with tilt.

**Comment by jacinyan** [comment]
> @jacinyan Yes, in the editor module. You'll want to trace through the code relating to the viewport navigation and specifically the tilt, which is the closest to flip. You can start tracing from an entry point such as `navigate_tool.rs` which deals with tilt.
> 
> That said, there are other issues that may be more straightforward and also higher priority (and thus more likely to receive bandwidth from me for code review).

Sure, I'll go with something more straightforward.

**Comment by Keavon** [comment]
@Sidharth-Singh10 can you please comment in this issue so I'm able to assign it to you? Thanks!

**Comment by Sidharth-Singh10** [comment]
#2394 fixes this
